Makefile

```diff
@@ -83,6 +84,27 @@ update-js-deps: $(js_deps)
 clean-js-deps:
 	rm -Rf $(js_deps)
 
+##------------
+## Tests
+##------------
+
+.PHONY: test-lint
+test-lint:             ## Run php unit tests
+test-lint:
+	find . -name \*.php -exec php -l "{}" \;
+
+.PHONY: test-php-unit
+test-php-unit:             ## Run php unit tests
+test-php-unit: vendor/bin/phpunit
+	pwd
+	make
+	./vendor/bin/phpunit --configuration ./src/Tests/phpunit.xml
+
+.PHONY: coverage-cache
+coverage-cache: vendor/bin/phpunit
+	sh -c "wget https://scrutinizer-ci.com/ocular.phar"
+	sh -c "php ocular.phar code-coverage:upload --format=php-clover clover.xml"
```

Comment:
And we don't need to bother with this - scrutinizer is not used any more. After this PR is finished, we can work on getting SonarCloud coverage reported.

Reply:
okay. thankyou